examples/flows/standard/flow-with-assistant/get_temperature.py

```diff
@@ -0,0 +1,16 @@
+import random
+from promptflow import tool
+
+
+@tool
+def get_temperature(location: str, unit: str = "c"):
+    """Estimate the current temperature of a given location. The temperature is randomly generated.
+    
+    :param location: Location to get the estimated temperature for.
+    :type location: str
+    :param unit: The unit of the temperature, either 'c' for Celsius or 'f' for Fahrenheit.
+                 Defaults to Celsius ('c').
+    :type unit: str
+    """
```

Comment:
Please add some random sleep (~1s) here to make the trace chart more meaningful.